script.js

```diff
@@ -0,0 +1,23 @@
+var Counter = React.createClass({
+    getInitialState: function() {
+        return {
+            counter: 0
+        };
+    },
+
+    decrement: function() {
+        this.setState({
+            counter: this.state.counter - 1
+        });
+    },
+
+    render: function() {
+        return React.createElement('div', {onClick: this.decrement},
+            React.createElement('span', {}, 'Licznik ' + this.state.counter)
```

Comment:
between two React elements you should add a colon `,`, because this is an object ;)